package.json

```diff
@@ -17,9 +19,6 @@
     "postversion": "git push && git push --tags && npm publish",
     "postinstall": "patch-package"
   },
-  "browserify-shim": {
-    "three": "global:THREE"
-  },
```

Comment:
You will need to define three as external if webpack is using three-to-ammo as ES6 module that itself import from three. See the webpack config in aframe-inspector https://github.com/aframevr/aframe-inspector/blob/e8798c73b6abd51566107e21a007f6fe90fbb5ad/webpack.config.js#L21-L24